examples/collector-exporter-node/docker/collector-config.yaml

```diff
@@ -0,0 +1,23 @@
+receivers:
+  otlp:
+    endpoint: 0.0.0.0:55678
+    cors_allowed_origins:
+      - http://localhost:8090
+  otlp/cors:
```

Comment:
i don't understand why you are declaring a second receiver which isn't used inside the pipeline ?

Reply:
some leftovers from testing